README.md

```diff
@@ -65,11 +65,12 @@ Other features of YAML are supported:
 ### Printing (untested)
 
 ```scala
-import io.circe.yaml.printer._
+import io.circe.yaml._
+import io.circe.yaml.syntax._
 
-val json = io.circe.parser.parse("""{"foo":"bar"}""").valueOr(throw _)
+val json = io.circe.parser.parse("""{"foo":"bar"}""").fold(throw _, identity)
```

Comment:
This is nitpicky, but either importing `cats.syntax.either._` and sticking with `valueOr` or using `.right.get` (or even `val Right(json) = ...`) seems preferable here—the `fold` and `identity` just seem like noise.